app/mailers/alaveteli_pro/metrics_mailer.rb

```diff
@@ -0,0 +1,35 @@
+module AlaveteliPro
+  ##
+  # A mailer responsible for sending out the weekly metrics report to
+  # the Pro contact
+  #
+  class MetricsMailer < ApplicationMailer
+    require_relative '../../../lib/alaveteli_pro/metrics_report.rb'
+
+    def self.send_weekly_report
+      weekly_report(AlaveteliPro::MetricsReport.new.report_data,
+                    pricing_enabled?).deliver_now
+    end
+
+    def weekly_report(data, pricing_enabled = false)
+      @data = data
+      @pricing_enabled = pricing_enabled
+
+      set_auto_generated_headers
+
+      subject = _("{{pro_site_name}} Weekly Metrics",
+                  pro_site_name: AlaveteliConfiguration.pro_site_name.html_safe)
+
+      mail(from: pro_contact_from_name_and_email,
+           to: pro_contact_from_name_and_email,
+           subject: subject
+          )
+    end
+
+    private
+
+    def self.pricing_enabled?
```

Comment:
The intent was to avoid an unexpected false positive when running this from the command line with `runner` but ... I'm no longer convinced it's correct.

The problem was that it was returning `nil` as `#<ActionMailer::Base::NullMail:0x000055c07a5dfeb8>` which is an object so it ends up registering as `true`. I'll loop back to this after restating it [as suggested below](https://github.com/mysociety/alaveteli/pull/5279#discussion_r300706207)